<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dot">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>370</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>310</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_0">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_6">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_minus">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>310</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_9">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_plus">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>310</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_equal">
    <property name="geometry">
     <rect>
      <x>359</x>
      <y>370</y>
      <width>181</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_multiply">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>250</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_divide">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>110</y>
      <width>67</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculator</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_browser">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="text_result">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>261</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
